<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NeatSpecification</class>
 <widget class="QWidget" name="NeatSpecification">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>599</width>
    <height>551</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-86</y>
        <width>571</width>
        <height>530</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <layout class="QFormLayout" name="formLayout">
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="PMLabel">
           <property name="text">
            <string>POPULATION_MAX</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="DC1Label">
           <property name="text">
            <string>DISTANCE_CONST_D</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="DC2Label">
           <property name="text">
            <string>DISTANCE_CONST_E</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="D3Label">
           <property name="text">
            <string>DISTANCE_CONST_W</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="DTLabel">
           <property name="text">
            <string>DISTANCE_THRESHOLD</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="POWCLabel">
           <property name="text">
            <string>PERCENTAGE_OFFSPRING_WITHOUT_CROSSOVER</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="PIMLabel">
           <property name="text">
            <string>PROBABILITY_INTERSPACIES_MATING</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="SPPANNLabel">
           <property name="text">
            <string>SMALLER_POPULATIONS_PROBABILITY_ADDING_NEW_NODE</string>
           </property>
          </widget>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="SPPANCLabel">
           <property name="text">
            <string>SMALLER_POPULATIONS_PROBABILITY_ADDING_NEW_CONNECTION</string>
           </property>
          </widget>
         </item>
         <item row="9" column="0">
          <widget class="QLabel" name="LPPANNLabel">
           <property name="text">
            <string>LARGER_POPULATIONS_PROBABILITY_ADDING_NEW_NODE</string>
           </property>
          </widget>
         </item>
         <item row="10" column="0">
          <widget class="QLabel" name="LPPANCLabel">
           <property name="text">
            <string>LARGER_POPULATIONS_PROBABILITY_ADDING_NEW_CONNECTION</string>
           </property>
          </widget>
         </item>
         <item row="11" column="0">
          <widget class="QLabel" name="PEADCLabel">
           <property name="text">
            <string>PROB_ENABLE_AN_DISABLE_CONNECTION</string>
           </property>
          </widget>
         </item>
         <item row="12" column="0">
          <widget class="QLabel" name="LPDLabel">
           <property name="text">
            <string>LARGE_POPULATION_DISCRIMINATOR</string>
           </property>
          </widget>
         </item>
         <item row="13" column="0">
          <widget class="QLabel" name="GLabel">
           <property name="text">
            <string>GENERATIONS</string>
           </property>
          </widget>
         </item>
         <item row="14" column="0">
          <widget class="QLabel" name="PCWLabel">
           <property name="text">
            <string>PROBABILITY_CHANGE_WEIGHT</string>
           </property>
          </widget>
         </item>
         <item row="15" column="0">
          <widget class="QLabel" name="PCNFPNLabel">
           <property name="text">
            <string>PROBABILITY_CHANGE_NODE_FUNCTION_PER_NODE</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="PMSpinBox">
           <property name="maximum">
            <number>1000000</number>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QDoubleSpinBox" name="DC1DoubleSpinBox">
           <property name="decimals">
            <number>7</number>
           </property>
           <property name="maximum">
            <double>1000000.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.000001000000000</double>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QDoubleSpinBox" name="DC2DoubleSpinBox">
           <property name="decimals">
            <number>7</number>
           </property>
           <property name="maximum">
            <double>1000000.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.000001000000000</double>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QDoubleSpinBox" name="DC3DoubleSpinBox">
           <property name="decimals">
            <number>7</number>
           </property>
           <property name="maximum">
            <double>1000000.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QSpinBox" name="DTSpinBox">
           <property name="maximum">
            <number>1000000</number>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QDoubleSpinBox" name="PIMDoubleSpinBox">
           <property name="decimals">
            <number>7</number>
           </property>
           <property name="maximum">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QDoubleSpinBox" name="SPPANNDoubleSpinBox">
           <property name="decimals">
            <number>7</number>
           </property>
           <property name="maximum">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="8" column="1">
          <widget class="QDoubleSpinBox" name="SPPANCDoubleSpinBox">
           <property name="decimals">
            <number>7</number>
           </property>
           <property name="maximum">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="9" column="1">
          <widget class="QDoubleSpinBox" name="LPPANNDoubleSpinBox">
           <property name="decimals">
            <number>7</number>
           </property>
           <property name="maximum">
            <double>1.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.000001000000000</double>
           </property>
          </widget>
         </item>
         <item row="10" column="1">
          <widget class="QDoubleSpinBox" name="LPPANCDoubleSpinBox">
           <property name="decimals">
            <number>7</number>
           </property>
           <property name="maximum">
            <double>1.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.000001000000000</double>
           </property>
          </widget>
         </item>
         <item row="11" column="1">
          <widget class="QDoubleSpinBox" name="PEADCDoubleSpinBox">
           <property name="decimals">
            <number>7</number>
           </property>
           <property name="maximum">
            <double>1.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.000001000000000</double>
           </property>
          </widget>
         </item>
         <item row="12" column="1">
          <widget class="QSpinBox" name="LPDSpinBox"/>
         </item>
         <item row="13" column="1">
          <widget class="QSpinBox" name="GSpinBox">
           <property name="maximum">
            <number>1000000</number>
           </property>
          </widget>
         </item>
         <item row="14" column="1">
          <widget class="QDoubleSpinBox" name="PCWDoubleSpinBox">
           <property name="decimals">
            <number>7</number>
           </property>
          </widget>
         </item>
         <item row="15" column="1">
          <widget class="QDoubleSpinBox" name="PCNFPNDoubleSpinBox">
           <property name="decimals">
            <number>7</number>
           </property>
          </widget>
         </item>
         <item row="16" column="0">
          <widget class="QCheckBox" name="CPPNCheckBox">
           <property name="text">
            <string>Enable CPPN functions</string>
           </property>
          </widget>
         </item>
         <item row="17" column="0">
          <widget class="QCheckBox" name="RecurrentConnectionCheckBox">
           <property name="text">
            <string>Enable Recurrent Connections</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QDoubleSpinBox" name="POWCDoubleSpinBox">
           <property name="decimals">
            <number>7</number>
           </property>
           <property name="maximum">
            <double>1.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.000001000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>File Management</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="SavePushButton">
            <property name="text">
             <string>Save</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="LoadPushButton">
            <property name="text">
             <string>Load</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
